lib/pavilion/series/info.py

```diff
@@ -149,21 +149,30 @@ def errors(self) -> int:
 
         status_obj = self._get_status_file()
 
+        error_values = [
+            'ERROR',
+            'CANCELLED',
+            'TIMEOUT',
+            ]
+
         errors = 0
         for status in status_obj.history():
-            if status.state in (status_file.SERIES_STATES.ERROR,
-                                status_file.SERIES_STATES.BUILD_ERROR,
-                                status_file.SERIES_STATES.CREATION_ERROR,
-                                status_file.SERIES_STATES.KICKOFF_ERROR):
-                errors += 1
+            for error_val in error_values:
+                if error_val in status.state:
```

Comment:
Using str.__contains__ here feels a bit kludgey and error-prone. A better solution would be to implement a mechanism to separate statuses into ERROR, CANCELLED, and TIMEOUT classes, either by:

a) Creating something like a `class` field on TestStatusInfo that takes on an enum value; or

b) By implementing three static methods that return sets of `ERROR`, `CANCELLED`, and `TIMEOUT` statuses respectively and then checking whether status.state is contained within each set (or simply the union of the three sets).

Reply:
The second option feels cleaner to me.